main.cpp

```diff
@@ -411,8 +405,12 @@ class NetworkService {
      * @returns 0 on success, negative value otherwise.
      */
     int init(Lwm2mConfig &config) {
+        SocketAddress sa;
+        nsapi_error_t err;
+
         NetworkInterface *netif = NetworkInterface::get_default_instance();
         if (!netif) {
+            printf("ERROR - can't get default network instance!");
```

Comment:
Adding a line ending (`\r\n`) would be nice.

Reply:
Duly noted, original commit fixed and force pushed.